projects/m1/039-random-password/js/index.js

```diff
@@ -0,0 +1,14 @@
+function generatePass(){
+    let passLenght = Math.floor(Math.random() * (10 - 7 + 1) + 7);
+    let minAscii = 33;
+    let maxAscii = 126;
+    const newPass = [];
+    for (i=0; i<passLenght;i++){
+        let asciiCode = Math.floor(Math.random() * (maxAscii - minAscii + 1)) + minAscii;
+        let character = String.fromCharCode(asciiCode);
+        newPass.push(character)
+    }
+    console.log(newPass.join(""));
```

Comment:
When you write a function it's **not a good idea** to **print inside** it, but it's ok to **return** values ​​and print them outside. 
For example:
`return newPass.join("")`
Outside:
`const password = generatePass();`
`console.log(password)`